src/libraries/System.Formats.Cbor/tests/System.Formats.Cbor.Tests.csproj

```diff
@@ -2,7 +2,14 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <nullable>enable</nullable>
+    <CborPropertyTests>false</CborPropertyTests>
```

Comment:
Who sets this property to true?

Reply:
I believe it is to have them disabled by default for now and if you want to run them you pass it globally as true.

Reply:
Correct, I use `dotnet test -p:CborPropertyTests=true`

Reply:
Why not running this in CI?

Reply:
The property tests are long running (adding ~10 seconds to test runtime). So if we want to run on CI we should either put them on the outer loop or we could just reduce the number of generated inputs per property test (currently set to 10K).